src/SFA.DAS.RoatpOversight.Web/Views/Shared/_AppealOutcomeDetails.cshtml

```diff
@@ -0,0 +1,88 @@
+﻿@using SFA.DAS.AdminService.Common.Extensions
+@using SFA.DAS.RoatpOversight.Web.Extensions
+@using SFA.DAS.RoatpOversight.Domain
+@model SFA.DAS.RoatpOversight.Domain.AppealDetails
+@{
+    var appealOutcomeTagClass = string.Empty;
+    var appealStatus = string.Empty;
+    bool showApplicationSectionOutcome = false;
+
+
+    if (Model.Status == AppealStatus.Unsuccessful)
+    {
+        appealOutcomeTagClass = "das-tag das-tag--solid-red";
+        appealStatus = Model.Status;
+    }
+    else if (Model.Status == AppealStatus.Successful || Model.Status == AppealStatus.SuccessfulFitnessForFunding || Model.Status == AppealStatus.SuccessfulAlreadyActive)
+    {
+        appealOutcomeTagClass = "das-tag das-tag--solid-green";
+        appealStatus = AppealStatus.Successful;        
+        if (Model.Status == AppealStatus.SuccessfulAlreadyActive || Model.Status == AppealStatus.SuccessfulFitnessForFunding)
+        {
+            showApplicationSectionOutcome = true;
+        }
+    }
+    else
+    {
+        appealOutcomeTagClass = "govuk-tag das-tag";
+        appealStatus = Model.Status;
+    }
+}
+<div class="govuk-form-group">
+    <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+            <h2 class="govuk-heading-l govuk-!-margin-bottom-3">Appeal outcome</h2>
+        </legend>
+        <dl class="govuk-summary-list">
+            <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">
+                Outcome
+            </dt>
+            <dd class="govuk-summary-list__value">
+                <strong class="@appealOutcomeTagClass">
+                    @appealStatus
+                </strong>
+            </dd>
+</div>
+@if (showApplicationSectionOutcome)
+{
+    <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+            Application Outcome
+        </dt>
+        <dd class="govuk-summary-list__value">
+            @Model.Status
+        </dd>
+    </div>
+}
+<div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+        Appeal determined date
+    </dt>
+    <dd class="govuk-summary-list__value">
+        @Model.AppealDeterminedDate.ToSfaShortDateString()
+    </dd>
+</div>
+<div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+        Outcome made by
+    </dt>
+    <dd class="govuk-summary-list__value">
+        @Model.UserName
+    </dd>
+</div>
+@if (appealStatus == AppealStatus.Successful)
+{
+    await Html.RenderPartialAsync("_AppealSuccessComments.cshtml", Model);
+}
+else if(appealStatus == AppealStatus.InProgress)
+{
+    await Html.RenderPartialAsync("_AppealInprogressComments.cshtml", Model);
+}
+else
+{
+    await Html.RenderPartialAsync("_AppealOutcomeComments.cshtml", Model);
+}
```

Comment:
Html.RenderPartialAsync is not MVC core compliant, please use the appropriate tag helper

Reply:
done

Reply:
added partial tag <parrtial> instead Html.PartialAsync , as partialAsync will not render for conditional based things

Reply:
@martin-lush